lib/init/grass.py

```diff
@@ -1373,19 +1374,11 @@ def set_language(grass_config_dir):
     encoding = None
 
     # Override value is stored in wxGUI preferences file.
-    # As it's the only thing required, we'll just grep it out.
-    try:
-        fd = open(os.path.join(grass_config_dir, "wx"), "r")
-    except:
-        # Even if there is no override, we still need to set locale.
-        pass
-    else:
-        for line in fd:
-            if re.search("^language", line):
-                line = line.rstrip(" %s" % os.linesep)
-                language = "".join(line.split(";")[-1:])
-                break
-        fd.close()
+    with open(os.path.join(grass_config_dir, "wx.json"), "r") as json_file:
```

Comment:
Will fail if if wx.json does not exist like at a first start of GRASS by a user. Perhaps an additional/larger try-except with whatever open raises is the way to go.

Reply:
Thanks for catching this!